### PR TITLE
Do not load index.php file in catalogue

### PR DESCRIPTION
**Pull request by maximebiloe**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.0.x
| Description?  | Do not load index.php file in catalogue
| Type?         | bug fix 
| Category?     | CO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-2283 (partially) Need a fix on TranlationToolsBundle too (https://github.com/PrestaShop/TranslationToolsBundle/pull/28)
| How to test?  |

### CODE REVIEW

**Comment by aleeks** [comment]
Thank you @maximebiloe

File: src/PrestaShopBundle/Translation/Provider/TranslationFinderTrait.php [hunk 1]
```diff
@@ -54,6 +54,10 @@ public function getCatalogueFromPaths($paths, $locale, $pattern = null)
         }
 
         foreach ($translationFiles as $file) {
+            if ('index.php' === $file->getFileName()) {
```
Comment (Quetzacoalt91) [inline src/PrestaShopBundle/Translation/Provider/TranslationFinderTrait.php]: Did you know that you could directly exclude this files from the finder execution ?

http://symfony.com/doc/current/components/finder.html#file-name

You could have written
```php
$translationFiles = $finder->files()->notName('index.php')->in($paths);
```

Comment (maximebiloe) [inline src/PrestaShopBundle/Translation/Provider/TranslationFinderTrait.php]: Arf 🤕  I didn't know.

Comment (Quetzacoalt91) [inline src/PrestaShopBundle/Translation/Provider/TranslationFinderTrait.php]: Refacto time ! 😬

Comment (maximebiloe) [inline src/PrestaShopBundle/Translation/Provider/TranslationFinderTrait.php]: Here it is @Quetzacoalt91 https://github.com/PrestaShop/PrestaShop/pull/7411

**Pull request merged**